### DISCUSSION TITLE
Convert Less to Sass

### DISCUSSION THREAD

**Discussion by vandroaurelio** [rsuite/rsuite#2094] [Ideas]
Sass is more popular than Less and widely used. If React Suite's styling was written in Sass it would make integration with other libraries easier. Please consider converting Less code into Sass.

**Comment by SevenOutman** [comment]
Hi @vjwolfa , thanks for your advice. Migrating to Sass is time-consuming and likely to have great impact on existing users. We would be cautious before we find the necessity and bring it into a plan.

**Comment by BossBele** [comment]
Having trouble with using less in NextJS 13. Sass would make a difference

**Comment by abdullaev388** [comment]
css itself is very powerful nowadays,rsuite creators should also consider using just plain css

**Reply by SevenOutman** [reply]
We're making some experimental changes around styles in v6. Stay tuned!
